codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionDecoder.java

```diff
@@ -492,13 +476,12 @@ public void onGoAwayRead(ChannelHandlerContext ctx, int lastStreamId, long error
         public void onWindowUpdateRead(ChannelHandlerContext ctx, int streamId, int windowSizeIncrement)
                 throws Http2Exception {
             Http2Stream stream = connection.requireStream(streamId);
-            verifyGoAwayNotReceived();
-            if (stream.state() == CLOSED || shouldIgnoreFrame(stream, false)) {
-                // Ignore frames for any stream created after we sent a go-away.
+
+            if (stream.state() == CLOSED || streamCreatedAfterGoAwaySent(stream)) {
```

Comment:
@buchgr - Now that we allow closed streams in the flow control window I was thinking we should processes this.  However I don't think it matters since these streams won't be sending any flow controlled frames in this state....unless extension frames do something weird... @nmittler - any thoughts...deal with it when we get to extension frames?